README.md

```diff
@@ -15,7 +15,12 @@
 #### Version support
 
 * Erlang - Prebuilt packages (17.5, 17.4, etc)
-* Elixir - Prebuilt releases (1.0.4, 1.0.3, etc) or prebuilt branches (master, stable, etc)
+  * The full list of prebuilt packages can be found here: https://github.com/HashNuke/heroku-buildpack-elixir-otp-builds/blob/master/otp-versions
```

Comment:
Can you add a note that if a version is missing you can send a PR that adds it.

Reply:
Added!